network/websocket.ksy

```diff
@@ -0,0 +1,77 @@
+meta:
+  id: websocket_dataframe
```

Comment:
This should match file name. Could you please rename the file to `websocket_dataframe.ksy`?